Resources/Storage/StorageFactory.py

```diff
@@ -35,6 +35,12 @@ def __init__( self, useProxy = False, vo = None ):
         self.vo = result['Value']
       else:
         RuntimeError( "Can not get the current VO context" )
+    self.remotePlugins = []
```

Comment:
These are already initialized in getStorages. Can it be initialized in one place only ? The __init__ is probably the best one